doc/source/contributing.rst

```diff
@@ -207,6 +206,26 @@ See the full conda docs `here <http://conda.pydata.org/docs>`__.
 
 At this point you can easily do an *in-place* install, as detailed in the next section.
 
+.. _contributing.windows:
+
+Creating a Windows development environment
+------------------------------------------
+
+To build on windows, you need to have compilers installed to build the extensions. You will need to install the appropriate Visual Studio compilers, VS 2008 for Python 2.7, and VS2012 for Python 3.5.
```

Comment:
VS 2010 for Python 3.2-3.4, VS 2015 for Python 3.5+

For VS2008, users may install either mingw, or the MS VC++ compiler for Python: https://www.microsoft.com/en-us/download/details.aspx?id=44266

For VS2010, users may install the Windows 7.1 SDK.